Composer/packages/client/src/pages/botProject/create-publish-profile/PublishProfileDialog.tsx

```diff
@@ -38,6 +38,17 @@ const Page = {
   ConfigProvision: Symbol('config'),
 };
 
+const formatDialogTitle = (current) => {
```

Comment:
Please change ```current``` here to ```isEditing: boolean```

Reply:
Will need to pick this up in a future PR.